File: verify/v2/verification_check/create-default/create-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

//Either a 'To' number or 'VerificationSid' must be specified
$options = [];
$options['verificationSid'] = 'VE4db3.....';
$verification_check = $twilio->verify->v2->services("VAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                         ->verificationChecks
                                         ->create("code", $options);

print($verification_check->sid);

File: verify/v2/verification_check/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

verification_check = @client.verify
                            .services('VAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                            .verification_checks
                            .create(to: '+14159373912', code: '1234')

puts verification_check.sid
